plugins/grpc/stats/client_handler.go

```diff
@@ -168,7 +168,7 @@ func (ch clientHandler) handleRPCEnd(ctx context.Context, s *stats.End) {
 	if s.Error != nil {
 		errorCode := s.Error.Error()
 		ts := tags.FromContext(ctx)
-		newTs := tags.NewTagSet(ts,
+		newTs := tags.NewSet(ts,
```

Comment:
This for me (no Go person at all) does not make sense too much, I prefer the previous version.

Reply:
Read my other argument.